source_test.go

```diff
@@ -778,6 +821,16 @@ func getSourceOp(ctx context.Context, t *testing.T, src Source) []*pb.Op {
 		return &st, nil
 	}
 
+	if len(fakeEntries) != 0 {
+		sOpt.GetFS = func(st llb.State) fs.ReadDirFS {
```

Comment:
I think this should always be set, even if there are no entries. If there are no entries, `fs.entries` can just be an empty array

Reply:
sounds good